lisa/sut_orchestrator/libvirt/transformers.py

```diff
@@ -234,7 +251,13 @@ def install(self) -> str:
         linux: Linux = cast(Linux, node.os)
         packages_list = self._distro_package_mapping[type(linux).__name__]
         self._log.info(f"installing packages: {packages_list}")
+        if isinstance(node.os, CBLMariner):
+            linux.install_packages(
+                ["mariner-repos-preview.noarch", "mariner-repos-extended"]
+            )
         linux.install_packages(list(packages_list))
+        username = node.tools[Whoami].get_username()
+        node.execute(f"usermod -aG {username} qemu", shell=True, sudo=True)
```

Comment:
The last two function call should be moved to user mod as suggested above

Reply:
Done.